### PR TITLE
Adding NOvA Project Proposal

### PR DESCRIPTION
**Pull request by FernandaPsihas**
Adding project plan "Tensorflow-TMVA GAN Interface Applied to NOvA Neutrino Events"

### CODE REVIEW

File: _activities/gsoc.md [hunk 1]
```diff
@@ -30,6 +30,7 @@ Instructions for participating projects and mentors can be found [here](/gsoc/gu
 | ![HSF](/images/hsf_logo_angled.png){:width="100px"} | The HEP Software Foundation (HSF) facilitates coordination and common efforts in high energy physics (HEP) software and computing internationally. [List of proposals](/gsoc/project_HSF.html)|
 | ![ATLAS](/images/ATLAS-Logo-Ref-RGB.png){:width="100px"} | [ATLAS](https://atlas.web.cern.ch/Atlas/Collaboration/) is a high-energy physics experiment at the LHC. [List of proposals](/gsoc/project_ATLAS.html)|
 | ![CMS](/images/CMS-Color.gif){:width="100px"} | [CMS](http://cms.web.cern.ch/) (Compact Muon Solenoid) is a high-energy physics experiment at the LHC. [List of proposals](/gsoc/project_CMS.html)|
+| ![NOvA](/images/NOvA_logo.png){:width="100px"} | [NOvA](https://www-nova.fnal.gov)(NOvA Off-Axis Electron Neutrino Appearance) is a Neutrino Oscillation experiment at Fermilab [List of proposals](/gsoc/project_NOvA.html)|
```
Comment (jouvin) [inline _activities/gsoc.md]: period missing after `Fermilab` (before `[List of proposals]`.

File: gsoc/project_NOvA.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+---
+project: NOvA
+layout: default
+description: |
+[NOvA](https://www-nova.fnal.gov) is a neutrino oscillation experiment whose measurement of this particle's properties relies on efficient identification and robust reconstruction of events. NOvA has already shown the first use of CNNs (convolutional neural networks) for [event classification] (http://inspirehep.net/record/1444342) in a HEP result. Now we are expanding the program of machine learning applications on NOvA events for their use in full event reconstruction and eventually event simulation applications.
```
Comment (jouvin) [inline gsoc/project_NOvA.md]: This line must be indented (2 spaces) else it is not recognized by Jekyll... Ideally you would like to split this very long line into multiple lines all indented the same way...

**Comment by jouvin** [comment]
Also, I don't have a clear picture about the status of GSoC projects... Is it still possible to add projects? Have you discussed it with people in charge of GSoC? @hegner any comment?

**Comment by jouvin** [comment]
@FernandaPsihas you also need to look at test failures related to URLs in your proposal that cannot be resolved...

**Comment by jouvin** [comment]
@FernandaPsihas looking in more details at the test error, this is a bit tricky... This is caused by the fact you added several organization in your front-matter section that doesn't exist as organization in the HSF GSoC site... Thing listed here must exist as a file under the `gsoc/` directory.

File: gsoc/project_NOvA.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+---
+project: NOvA
+layout: default
+description: |  [NOvA](https://www-nova.fnal.gov) is a neutrino oscillation experiment whose measurement of this particle's properties relies on efficient identification and robust reconstruction of events. NOvA has already shown the first use of CNNs (convolutional neural networks) for [event classification] (http://inspirehep.net/record/1444342) in a HEP result. Now we are expanding the program of machine learning applications on NOvA events for their use in full event reconstruction and eventually event simulation applications.
```
Comment (jouvin) [inline gsoc/project_NOvA.md]: Unfortunately, this is still incorrect... You should keep `[NOvA]....` on a separate line. Look at `gsoc/project_SixTrack.md` for an example.

**Comment by FernandaPsihas** [comment]
I think all the changes you requested have been pushed. Did I miss anything?

I’m cc’ing Sergei Gleyzer who can confirm that this addition has been discussed with the admins.

Cheers,

----------------------------------------------
Fernanda Psihas
Graduate Research Assistant
Department of Physics
Indiana University
psihas@fnal.gov<mailto:psihas@fnal.gov>
----------------------------------------------

"Your focus determines your reality."
                                            - Qui-Gon Jin



On Apr 18, 2017, at 5:07 PM, Michel Jouvin <notifications@github.com<mailto:notifications@github.com>> wrote:


@jouvin requested changes on this pull request.

________________________________

In gsoc/project_NOvA.md<https://github.com/HEP-SF/hep-sf.github.io/pull/139#discussion_r112064306>:

> @@ -0,0 +1,7 @@
+---
+project: NOvA
+layout: default
+description: |  [NOvA](https://www-nova.fnal.gov) is a neutrino oscillation experiment whose measurement of this particle's properties relies on efficient identification and robust reconstruction of events. NOvA has already shown the first use of CNNs (convolutional neural networks) for [event classification] (http://inspirehep.net/record/1444342) in a HEP result. Now we are expanding the program of machine learning applications on NOvA events for their use in full event reconstruction and eventually event simulation applications.


Unfortunately, this is still incorrect... You should keep [NOvA].... on a separate line. Look at gsoc/project_SixTrack.md for an example.

**Comment by jouvin** [comment]
@FernandaPsihas the font-matter syntax error is fixed, I confirm. But you missed my [comment](https://github.com/HEP-SF/hep-sf.github.io/pull/139#issuecomment-294981953) about the invalid organizations resulting into invalid links. Look at the [Travis check log](https://travis-ci.org/HEP-SF/hep-sf.github.io/builds/223414406).

**Comment by FernandaPsihas** [comment]
Hi Fernanda, all,

That's correct. Benedict, Michel - thanks a lot for your help with this.

Best regards,

Sergei

**Comment by FernandaPsihas** [comment]
@jouvin I think I fixed the issue on your previous comment but the build is still failing. Am I missing a change? 

Cheers,

Fernanda

**Comment by jouvin** [comment]
@FernandaPsihas you are right, you fixed all the problems! The failing test is due to an URL into an old page that is timing out... We are checking that the URLs in our site are reachable: unfortunately some external URLs are not reliable at all and this shows up when other mods are made. You are just a victim! I'll merge this PR and fix the problem with the problematic URL later.
Thanks for your contribution!

**Pull request merged**